compose/ui/ui/src/webTest/kotlin/androidx/compose/ui/input/TextInputTests.kt

```diff
@@ -78,70 +65,66 @@ class TextInputTests : OnCanvasTests {
         }
     }
 
-    private suspend fun InputInteractor.createTextFieldWithChannel(
-        content: @Composable () -> Unit = {
-            val focusRequester = focusRequesters[0]
+    private fun assertTextEquals(expected: String, actual: MutableState<TextFieldValue>, message: String? = null) {
```

Comment:
Small suggestion: make the `MutableState` the receiver.
```
textFieldValue.assertTextEquals("xxx")
```
looks a bit better IMO.

Reply:
@m-sasha make sense, let me to it

Reply:
@m-sasha pushed